addons/project/static/tests/project_project_form_view.test.js

```diff
@@ -18,3 +24,28 @@ test("project.project (form)", async () => {
     });
     expect(".o_form_view").toHaveCount(1);
 });
+
+test("project.project (form) show archive action for project manager", async () => {
```

Comment:
should we also have a similar case for the opposite case where the user is not a project  manager?